autoload/leaderf.vim

```diff
@@ -855,3 +855,9 @@ function! leaderf#setAmbiwidth(val) abort
     exec g:Lf_py "from leaderf.devicons import setAmbiwidth"
     exec g:Lf_py printf("setAmbiwidth('%s')", a:val)
 endfunction
+
+autocmd ColorScheme * call leaderf#highlightDevIcons()
```

Comment:
the autocmd can be put here:
https://github.com/Yggdroot/LeaderF/blob/e039584b52ec46fa7194eb8520efd7a76a34bb70/autoload/leaderf/python/leaderf/instance.py#L725-L737

Reply:
I fixed it.